sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
 # Release History
 
+## 11.3.0 (Unreleased)
```

Comment:
Two questions:
1. Do we want to port some beta fixes into this GA release?
2. Do you want to mention that some 11.3.0 beta features are not available in this release?